docs/tutorials/multimodal/document/index.rst

```diff
@@ -1,5 +1,35 @@
 Document Data
+=============
+
+Pre-requisite
 -------------
+
+Processing document data depends on the optical character recognition (OCR) package ``tesseract``.
+
+
+For Ubuntu users, you can install Tesseract and its developer tools by simply running:
```

Comment:
Can we link a general installation guide when users' platform doesn't fit into these category. Like this [one](https://github.com/tesseract-ocr/tesseract#installing-tesseract)